docs/level-9-moban-dependency-as-pypi-package/.moban.yml

```diff
@@ -2,10 +2,10 @@ requires:
   - pypi-mobans-pkg
 configuration:
   template_dir:
-    - "pypi-mobans-pkg:templates"
+    - "pypi://pypi-mobans-pkg/templates"
```

Comment:
if the `pypi` driver is to be a generic one, the syntax here would be "pypi://pypi-mobans-pkg/resources/templates"

I think that is worth it -- then the pypi driver can be used by other python software to refer to package data, e.g. instead of using [`pkgutil.get_data`](https://docs.python.org/2/library/pkgutil.html#pkgutil.get_data)

Reply:
good idea! then moban logic get simplified